src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -28,6 +30,7 @@
 class RequestDataCollector extends DataCollector implements EventSubscriberInterface, LateDataCollectorInterface
 {
     protected $controllers;
+    protected $sessionUsages = [];
```

Comment:
should be private (all new properties should start as private unless we decide to make them an inheritance-based extension point, which we try to avoid most of the time as they are a pain for BC)